### PR TITLE
Add download links to long files

### PR DESCRIPTION
**Pull request by nhennigan**
Added download links to longer files. I also replaced those files with an {literalinclude} so that they still appear in the docs the same way, but we only need to update one location in the future.

### CODE REVIEW

File: docs/src/assets/configuration.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+     network:
+       enabled: true
+     dns:
+       enabled: true
+     local-storage:
+       enabled: true
+   extra-node-kubelet-args:
+     --reserved-cpus: "0-31"
+     --cpu-manager-policy: "static"
+     --topology-manager-policy: "best-effort"
```
Comment (bschimke95) [inline docs/src/assets/configuration.yaml:11]: nit: please add a newline - otherwise GH complains with this red circle

**Review by bschimke95 — APPROVED** [review]
LGTM, only a minor nit

File: docs/src/assets/configuration.yaml (hunk 1 above)
Comment (nhennigan) [reply docs/src/assets/configuration.yaml]: Fixed thanks for pointing it out

**Comment by bschimke95** [comment]
thanks, feel free to merge if everything is addressed.

**Pull request merged**